### ISSUE TITLE
Criar interface para edição dos dados do dicionário

### ISSUE THREAD

**Issue by brunomaroneze** [brunomaroneze/dicbio#33]
Os arquivos CSV que contêm os dados do dicionário (DadosDoDicionario.csv e definitions.csv) estão sendo editados pelo Google Planilhas. Além de ficar ruim de visualizar, porque as células são pequenas para muito texto, ainda fico tendo que incluir os códigos HTML diretamente no texto (senão não dá para inserir quebra de linha, por exemplo). Ajudaria muito ter uma espécie de formulário, no estilo dos antigos formulários do Microsoft Access, em que fiquem visíveis as informações de uma única linha na tela, e para ver a próxima clicaríamos numa setinha.

**Comment by rodrigodca** [comment]
Estive testando uma ferramenta chamada Symphytum (https://github.com/giowck/symphytum) o qual tem exatamente essas premissas: Editar CSV e organizar e redimensionar os campos de texto de forma a caber blocos de texto. Segue print do csv sendo usado no momento.
![Capturar](https://user-images.githubusercontent.com/123764287/228056496-ec19bb4b-2266-4a3a-ae8a-5641eecd5bc6.JPG)

**Comment by brunomaroneze** [comment]
Sim, é isso mesmo que estou procurando! Tem que instalar? Ou dá para usar online?
